R/dsl.R

```diff
@@ -39,5 +39,18 @@ mcstate_dsl <- function(x, type = NULL) {
     x <- rlang::quo_get_expr(quo)
   }
   exprs <- dsl_preprocess(x, type)
+  dat <- dsl_parse(exprs)
```

Comment:
So for now, this is just to test dsl_parse and report errors / success - (dat not used yet)

Reply:
yeah, slowly building up to eventually generating something here